hmt_escrow_js/Job.js

```diff
@@ -14,7 +14,7 @@ class Job {
     this._manifest_url = manifest_url
     this._factory_addr = factory_addr
 
-    this.serialized_manifest = this._download_manifest()
+    this._serialized_manifest = this._download_manifest()
     this.amount = null
```

Comment:
Interesting, why does `amount` not have a `_` in front of, just like the others?

Reply:
stupidity on my part

Reply:
:) Then can I ask, why do all of them have a `_` in front of them? Are they internal in any way?

Reply:
habit for internals when designing classes, here the usecase is slightly different but I stuck with it